rpm/stackdriver-agent

```diff
@@ -19,13 +19,13 @@
 ### END INIT INFO
 
 # Source function library.
-if [[ -f /lib/lsb/init-functions ]]; then
-  . /lib/lsb/init-functions
-else
+if [[ -f /etc/init.d/functions ]]; then
```

Comment:
The breakage is actually in the implementation of the `status` command — it's still just as broken on systems that use `/lib/lsb/init-functions` (which doesn't define `status`).

Let's add a comment here that this is a workaround to unbreak legacy systems, not a fix.

Reply:
Added a comment. Please review (I'm not sure if it's accurate)